workbench-core/environments/src/interfaces/listEnvironmentTypeRequest.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { QueryParameterFilter, FilterRequest, SortRequest } from '@aws/workbench-core-base';
+
+export interface ListEnvironmentTypeRequest {
```

Comment:
nit: should environment type(S) be plural?

Reply:
changed to have plural